agent/test-agent-cli/src/get_secret.rs

```diff
@@ -0,0 +1,29 @@
+use crate::error::{Result, SecretKeyFetchSnafu};
+use crate::Client;
+use argh::FromArgs;
+use snafu::OptionExt;
+
+#[derive(Debug, FromArgs, PartialEq)]
+#[argh(subcommand, name = "get_secret", description = "Get secret for a key")]
+pub(crate) struct GetSecret {
```

Comment:
Is this tested anywhere?

Reply:
Yeah, I have added it in example test script